sound/soc/sof/probe.c

```diff
@@ -0,0 +1,205 @@
+// SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+//
+// Author: Cezary Rojewski <cezary.rojewski@intel.com>
+//
+
+#include "sof-priv.h"
+#include "probe.h"
+
+int sof_ipc_probe_init(struct snd_sof_dev *sdev,
+		size_t buffer_size)
```

Comment:
@crojewsk can these 2 lines be on the same line?

Reply:
You got it @ranj063 !
Ack.